testeng/jobs/countWorkers.groovy

```diff
@@ -1,36 +0,0 @@
-package testeng
-
-import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
-
-job('count-workers') {
-
-    logRotator JENKINS_PUBLIC_LOG_ROTATOR()
-    label('master')
-    concurrentBuild(false)
-
-    scm {
-        git {
-            remote {
-                url('https://github.com/edx/testeng-ci.git')
-            }
-            branch('*/master')
-            browser()
-        }
-    }
-
-    triggers{
-        cron('*/5 * * * *')
-    }
-
-    wrappers {
-        timestamps()
-    }
-
-    steps {
-        shell(readFileFromWorkspace('testeng/resources/count-workers.sh'))
```

Comment:
Can this shell script go away too?  It in turn seems to try running a Python script that I can't find.

Reply:
Ah yeah, great catch. It calls the script I am removing here: https://github.com/edx/testeng-ci/pull/163